start_notebook_dali.sh

```diff
@@ -1,58 +0,0 @@
-#!/bin/bash
-
-IMAGE_NAME=$1
-JUPYTER_TYPE=$2
-NOTEBOOK_DIR=$3
-
-IMAGE_DIR='/dali/lgrandi/xenonnt/singularity-images'
-
-# if we passed the full path to an image, use that
-if [ -e ${IMAGE_NAME} ]; then
-  CONTAINER=${IMAGE_NAME}
-# otherwise check if the name exists in the standard image directory
-elif [ -e ${IMAGE_DIR}/${IMAGE_NAME} ]; then
-  CONTAINER=${IMAGE_DIR}/${IMAGE_NAME}
-# if not any of those, throw an error
-else
-  echo "We could not find the container at its full path ${IMAGE_NAME} or in ${IMAGE_DIR}. Exiting."
-  exit 1
-fi
-
-if [ "x${JUPYTER_TYPE}" = "x" ]; then
-  JUPYTER_TYPE='lab'
-fi
-
-echo "Using singularity image: ${CONTAINER}"
-
-PORT=$(( 15000 + (RANDOM %= 5000) ))
-SINGULARITY_CACHEDIR=/dali/lgrandi/$USER/singularity_cache
-
-# script to run inside container
-INNER=/dali/lgrandi/$USER/.singularity_inner
```

Comment:
Why was this whole path instead of just being .singularity_inner?

Reply:
I don't know why people wrote it differently for dali. But I think it's better to be consistent here. As people can only start notebook with the dali partition on dali, setting INNER by PWD should be the best practice.